kafka-ballerina/tests/consumer_client_tests.bal

```diff
@@ -0,0 +1,1088 @@
+    int offsetValue = committedPartitionOffset.offset;
+
+    test:assertEquals(offsetValue, messageCount);
+}
+
+@test:Config {
+    dependsOn: [consumerServiceCommitTest]
+}
+function consumerServiceCommitOffsetTest() returns error? {
+    ConsumerConfiguration consumerConfiguration = {
+        topics: [topic5],
+        offsetReset: OFFSET_RESET_EARLIEST,
+        groupId: "consumer-service-commit-offset-test-group",
+        clientId: "test-consumer-4",
+        autoCommit: false
+    };
+    TopicPartition topicPartition = {
+        topic: topic5,
+        partition: 0
+    };
+    Listener serviceConsumer = check new (DEFAULT_URL, consumerConfiguration);
+    check serviceConsumer.attach(consumerServiceWithCommitOffset);
+    check serviceConsumer.'start();
+
+    int messageCount = 10;
+    int count = 0;
+    while (count < messageCount) {
+        check sendMessage(count.toString().toBytes(), topic5);
+        count += 1;
+    }
+    Consumer consumer = check new (DEFAULT_URL, consumerConfiguration);
+    ConsumerRecord[] consumerRecords = check consumer->poll(1);
+    PartitionOffset? committedOffset = check consumer->getCommittedOffset(topicPartition);
+    PartitionOffset committedPartitionOffset = <PartitionOffset>committedOffset;
+    int offsetValue = committedPartitionOffset.offset;
+
+    test:assertEquals(offsetValue, messageCount + 10);
+}
+//
+////@test:Config {}
+////function consumerServiceGracefulStopTest() returns error? {
+////    check sendMessage(TEST_MESSAGE.toBytes(), topic1);
+////    ConsumerConfiguration consumerConfiguration = {
+////        topics: [topic1],
+////        offsetReset: OFFSET_RESET_EARLIEST,
+////        groupId: "consumer-service-graceful-stop-test-group",
+////        clientId: "test-consumer-5"
+////    };
+////    Listener consumer = check new (DEFAULT_URL, consumerConfiguration);
+////    check consumer.attach(consumerService);
+////    check consumer.'start();
+////    //runtime:sleep(7);
+////    //test:assertEquals(receivedMessage, TEST_MESSAGE);
+////    check consumer.gracefulStop();
+////    //runtime:sleep(7);
+////    //check sendMessage(TEST_MESSAGE2.toBytes(), topic1);
+////    //test:assertNotEquals(receivedMessage, TEST_MESSAGE2);
+////}
+//
+////@test:Config {}
+////function consumerServiceDetachTest() returns error? {
+////    ConsumerConfiguration consumerConfiguration = {
+////        topics: [topic1],
+////        offsetReset: OFFSET_RESET_EARLIEST,
+////        groupId: "consumer-service-detach-test-group",
+////        clientId: "test-consumer-6"
+////    };
+////    Listener consumer = check new (DEFAULT_URL, consumerConfiguration);
+////    check consumer.attach(consumerService);
+////    check consumer.'start();
+////
+////    runtime:sleep(7);
+////    check consumer.detach(consumerService);
+////    check sendMessage(TEST_MESSAGE.toBytes(), topic1);
+////    runtime:sleep(7);
+////    test:assertEquals(receivedMessage, EMPTY_MESSAGE);
+////}
+//
```

Comment:
Why are we keeping these commented-out blocks? Shall we remove them, if we aren't using them?

Check the other places as well.